### PR TITLE
update Nodejs sdk

### PR DESCRIPTION
**Pull request by Achraf-El-khaier**
update nodejs sdk with voucher and loyalty

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ class LiteApi {
         this.apiKey = apiKey;
         this.serviceURL = "https://api.liteapi.travel/v3.0";
         this.bookServiceURL = "https://book.liteapi.travel/v3.0";
+        this.baseUrl = 'https://da.liteapi.travel';
```
Comment (squallstar) [inline index.js:6]: This would be "dashboardUrl", right?

**Review by squallstar — COMMENTED** [review]
See comment. 

I think you need to update the README file too to list out all the new functions available. 

Render it as markdown to see how it's structured.

File: index.js (hunk 1 above)
Comment (Achraf-El-khaier) [reply index.js:6]: Yes, and it is the URL used to get vouchers

Comment (squallstar) [reply index.js:6]: My question here is to change this variable name to `dashboardURL`

Comment (Achraf-El-khaier) [reply index.js:6]: ok  i will change this variable name from baseUrl to dashboardURL

**Review by squallstar — APPROVED** [review]

**Pull request merged**